Swerve2022/src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -97,19 +74,44 @@ public SwerveModuleState getState( )
   public void setDesiredState(SwerveModuleState desiredState)
   {
     // Optimize the reference state to avoid spinning further than 90 degrees
-    SwerveModuleState state = SwerveModuleState.optimize(desiredState, new Rotation2d(m_turningEncoder.get( )));
+    SwerveModuleState state = SwerveModuleState.optimize(desiredState, new Rotation2d(degToRad(m_turningCANCoder)));
 
     // Calculate the drive output from the drive PID controller.
-    final double driveOutput = m_drivePIDController.calculate(m_driveEncoder.getRate( ), state.speedMetersPerSecond);
+    final double driveOutput = m_drivePIDController.calculate(getSpeedMPS(m_driveMotor), state.speedMetersPerSecond);
 
     final double driveFeedforward = m_driveFeedforward.calculate(state.speedMetersPerSecond);
 
     // Calculate the turning motor output from the turning PID controller.
-    final double turnOutput = m_turningPIDController.calculate(m_turningEncoder.get( ), state.angle.getRadians( ));
+    final double turnOutput = m_turningPIDController.calculate(degToRad(m_turningCANCoder), state.angle.getRadians( ));
 
     final double turnFeedforward = m_turnFeedforward.calculate(m_turningPIDController.getSetpoint( ).velocity);
 
     m_driveMotor.setVoltage(driveOutput + driveFeedforward);
     m_turningMotor.setVoltage(turnOutput + turnFeedforward);
   }
+
+  public double getDistanceMeters(WPI_TalonFX driveMotor)
+  {
+    return nativeUnitsToMeters(driveMotor.getSelectedSensorPosition( ));
+  }
+
+  private double getSpeedMPS(WPI_TalonFX driveMotor)
+  {
+    return nativeUnitsToMPS(driveMotor.getSelectedSensorVelocity( ));
+  }
+
+  private double nativeUnitsToMeters(double nativeUnits)
+  {
+    return nativeUnits * DTConsts.kEncoderMetersPerCount;
+  }
+
+  private double nativeUnitsToMPS(double nativeUnitsVelocity)
+  {
+    return nativeUnitsVelocity * DTConsts.kEncoderMetersPerCount * 10;
+  }
+
+  private double degToRad(CANCoder encoder)
```

Comment:
Since this method is not just converting degrees to radians, let's change the name. Just like getDistanceMeters which takes a motor controller, this method is returning an angle in radians--so maybe getAngleRadians would be a better name.